src/lib.rs

```diff
@@ -481,7 +485,19 @@ mod test {
         assert_eq!(decoded_result.unwrap(), hash1);
     }  
 
-    #[cfg(feature = "bench")] 
+    #[test]
+    fn base64_error_on_empty() {
+        let decoded_result = ImageHash::from_base64("");
+        assert!(decoded_result.is_err());
+        /* FIXME: Figure out how to make the below run
+        match decoded_result {
+            Err(FromBase64Error::InvalidBase64Length) => assert!(true),
```

Comment:
The core problem I've got with the commented out lines is that they don't even compile. Any thoughts on how to fix that? I get
```
src/lib.rs:493:17: 493:53 error: unresolved enum variant, struct or const `InvalidBase64Length` [E0419]
src/lib.rs:493             Err(FromBase64Error::InvalidBase64Length) => (),
                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```